### DISCUSSION TITLE
Multiple slides with different colors on the pagination?

### DISCUSSION THREAD

**Discussion by Devotee007** [nolimits4web/swiper#5530] [Q&A]
I have a page that contains multiple slides, it has worked great. But now we want to have a slide on a different background and needs to change the colour of the pagination. Is it possible to have one slide with different colour and the rest in the colour we've used before?

**Comment by Arhell** [comment]
Hello, add a custom class and with css do it.

**Reply by Devotee007** [reply]
I have already used a custom class for the standard one, I need to have another class for the secondary colour. I think?

**Reply by Arhell** [reply]
yes, for example you have custom-1, add custom-1--second

**Reply by Devotee007** [reply]
Ah, I didn't know that was possible. Thank you! So in the below code I  add a second bulletClass and modifierClass?

`pagination: {
                    el: '.swiper-pagination',
                    dynamicBullets: true,
                    clickable: true,
                    bulletClass: 'custom-bullet-primary',
                    modifierClass: 'swiper-pagination-custom',
                },`

**Reply by Arhell** [reply]
Of course, the name is up to you and based on this class, you do what you need.

**Reply by Devotee007** [reply]
Yes, of course, but is this correct for example:
`
pagination: {
                    el: '.swiper-pagination',
                    dynamicBullets: true,
                    clickable: true,
                    bulletClass: 'custom-bullet-primary',
                    bulletClass: 'custom-bullet-secondary',
                    modifierClass: 'swiper-pagination-primary',
                    modifierClass: 'swiper-pagination-secondary',
                },`

**Reply by Arhell** [reply]
Maybe. I would put a custom class not on pagination, but on the slider itself (wrapper).

**Reply by Devotee007** [reply]
I loop out the different slides with this, there I add a custom class. Can I use that class to get the custom secondary colour? Sorry for these beginners questions.

const customSlider = document.querySelectorAll('.swiper');

        for( i=0; i< customSlider.length; i++ ) {
            customSlider[i].classList.add('swiper-container-' + i);

            const swiper = new Swiper('.swiper-container-' + i, {

                direction: 'horizontal',
                slidesPerView: 'auto',
                slidesPerGroupSkip: 1,
                
                grabCursor: false,
                speed: 500,
                centerInsufficientSlides: true,
                centeredSlides: false,
                loop: false,
                watchOverflow: true,
                shortSwipes: true,
                threshold:50,

                pagination: {
                    el: '.swiper-pagination',
                    dynamicBullets: true,
                    clickable: true,
                    bulletClass: 'custom-bullet-primary',
                    modifierClass: 'swiper-pagination-custom',
                },
            });
        }

**Reply by Arhell** [reply]
Check the output, if the class is pulled up, then everything is ok.

If not, then just initialize the 2nd slider
document.querySelectorAll('.swiper2');

The same settings with only an additional class

**Reply by Devotee007** [reply]
Do you mean I should, on the slider that I want to have different color, just add a second class and then use a copy of the above code and change the  const customSlider = document.querySelectorAll('.swiper'); with the new class and have a specific for loop for that one?  Or do you mean I should use the above code with the addition of an if statement like classList.contains?

**Reply by Arhell** [reply]
Not necessarily, you can just initialize 2 sliders without loops.

In html `<div class="swiper1">` ` document.querySelectorAll('.swiper1')`
initialize it and second `<div class="swiper2 customClass">` ` document.querySelectorAll('.swiper2')`

two initializations, that's fine.

**Reply by Devotee007** [reply]
Thanks I have two different initialisations now.  It works great. Will see if I can get them to the same initialization later. :)
